index.html

```diff
@@ -0,0 +1,129 @@
+              was just that good. They hoped and hoped that their own star players would someday become good enough to be able to challenge
+              his greatness and be able to beat him. Rarely has that worked, especially in playoff contests. They hated, but respected him because
+              the amount of time and energy he put in to strengthen his game and become the best player he could possibly be is unparalelled.
+              <br><br> Throughout this past season however, Kobe got way more cheers than jeers. Since he announced his retirement early in the season
+              fans began clamoring for tickets. The feel in most of the away stadiums seemed confusing for a while.
+              Fans didn't seem to know what they wanted to see most: their team win, or Kobe go off for 30 or 40 points? On some nights they got to see a Kobe
+              show but their team would beat the Lakers, which was ideal. But Kobe spent most of the first half of the season shooting bricks.
+              His legs were not under him, and his body was not used to the high level of play that an NBA game requires after missing almost two full seasons.
+              He started coming into his own midway through the season with a group of young players around him leading the show. But when Kobe had a chance, he still
+              showed flashes of his old self that people had become accustomed to throughout the years. <br><br> Fast forward to last Wednesday
+              night, and Kobe put on a masterpiece performance. He scored 60 Points for the sixth time in his career. Yes you read that right, SIXTY POINTS.
+              Many sports analysts guessed Kobe would go out and score about 30 to 40 points due to the amount of shots he would take. But
+              they never saw this coming. It was a performance for the ages. He had 22 points by halftime and scored 23 points in the 4th quarter
+              alone to lead a 10 point Laker comeback in the final two minutes of the game. Kobe looked like Kobe. Hitting fadeaway jumpshots,
+              three pointers, and a go-ahead jumper in the last minute of the game to put the Lakers ahead for good. After the ensuing Jazz miss,
+              Kobe was sent to the foul line for the last time in his career sitting on 58 points. He made his first and second free-throws to give
+              him the big 6-0. He then exited the game and addressed the crowd afterwards in an amazing post-game speech with the final words
+              being "Mamba Out". He was already legendary before this game, but now he has erased the memory of him struggling for most of this past season
+              and elevating himself into another stratosphere which most of us fans didn't even think was possible. </p>
+            <section class="article-body"></section>
+            <a href="#" class="read-on">Read on &rarr;</a>
+          </article>
+      </section>
+      <section id="blogs" class="section2">
```

Comment:
Ids are unique. You already used id="blogs", so you need something else for this section.